app/pb_kits/playbook/pb_title/title.rb

```diff
@@ -17,8 +17,8 @@ class Title
                  default: "h3"
       prop :text
       prop :variant, type: Playbook::Props::Enum,
-                    values: [nil, "primary"],
-                    default: nil
+                     values: [nil, "primary"],
```

Comment:
I wonder if this make sense in terms of other text color styles? For example, I believe they are called positive, negative, and link in sketch. Thoughts?